src/maps/russia.ru.js

```diff
@@ -0,0 +1,347 @@
+export default {
+  label: 'Россия',
+  viewBox: '0 0 990 593',
+  locations:[
+    {
+      name: 'Республика Карелия',
```

Comment:
If you have time, could you add an English version (russia.en)?
Else I will do it myself with google translate 😄